geometry_msgs/Pose2D pose
float64[36] covariance ## JS: use float64[9] since we use 3x3 cov matrixes